pyaedt/generic/DataHandlers.py

```diff
@@ -482,6 +489,39 @@ def to_aedt(code):
     return return_code
 
 
+def str_to_bool(s):
+    """Convert a ``"True"`` or ``"False"`` string to its corresponding Boolean value.
+
+    If the passed arguments are not relevant in the context of conversion, the argument
+    itself is returned. This method can be called using the ``map()`` function to
+    ensure conversion of Boolean strings in a list.
+
+    Parameters
+    ----------
+    s: str
+
+    Returns
+    -------
+    bool or str
+         The method is case-insensitive.
```

Comment:
```suggestion
         The method is case-insensitive.
         
```
The blank line is needed before a bulleted list for proper rending in the doc.